### DISCUSSION TITLE
Adding an email address to Github Account

### DISCUSSION THREAD

**Discussion by Abhijit-Iyer** [community/community#78803] [Actions]
### Select Topic Area

General

### Body

I've been trying to add a certain email ID to my Github account, but I'm getting an error stating 'Error adding <Email ID>: email is already in use. This appears even though I have not added this email ID to Github. What could be the reason?

**Comment by Kiran1689** [comment]
As the error says, That email id is already in use.. Check any existing GitHub accounts  you have created using that email id...

**Reply by Eh-Vee** [reply]
Right! I forgot I made a GitHub years ago, before I'd even gained an interest in coding. I just stumbled upon the site, made an account to check it out and got bored because I didn't know anything about coding 😂.

**Comment by lukaspestalozzi** [comment]
see https://github.com/orgs/community/discussions/73524

**Comment by Eh-Vee** [comment]
You should check to see if you have another account with that email. You can always do forgot password to sign in and delete the account if you really want to add the email to your current account.
